### ISSUE TITLE
Export function opens blank new tab, but doesn't download

### ISSUE THREAD

**Issue by ebenz99** [gabriellesc/piet-ide#5]
Experienced on Google Chrome on Mac

**Comment by razetime** [comment]
Yes, same problem here. How do I rectify this on a Mac? I managed to make a program but was unable to save it.

@gabriellesc just got into piet, would love to use this editor to write some programs and save them.

**Comment by ebenz99** [comment]
sadly, there wasn't a fix that I could find. I ended up writing my own (slightly different) version of Piet because as far as I could tell (at the time), there wasn't really a working editor or compiler for Mac. Here it is if you're interested: https://github.com/ebenz99/epiet

**Comment by razetime** [comment]
Thanks a lot. You're a lifesaver.

**Comment by dj-smith** [comment]
+1, Experiencing the same bug, Google Chrome & Microsoft Edge on Windows.
